pgx/src/spi.rs

```diff
@@ -578,7 +578,16 @@ impl<'a> SpiClient<'a> {
                     Some(pg_sys::SPI_tuptable)
                 }
             },
+            #[cfg(any(feature = "pg11", feature = "pg12"))]
```

Comment:
Some `// SAFETY` docs is all I can see we should do here.

Reply:
Done